docs/catalog/cloud/optimize-application-components-that-consome-the-most-resources.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Optimize application components that consume the most resources
+
+## Description
+
+Monitor the application to identify components that consume the most time or resources. Optimize the code or architecture so, that these component minimize the resource usage while maximizing performance.
+
+## Solution
+
+Cloud providers support monitoring by providing logs and other methods to inspect the performance of the application additionally to tools to inspect the resource usage. Utilize these tools to identify application components that use the most resources and optimize these.
+
+Use a code profiler for deep application component analysis and consider other algorithms for an improved performance or using algorithms implemented in hardware for less resource usage.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, optimize application components, will impact two parts:
+
+- `E`: If memory, amount of used CPUs or storage can be decreased, then the embodied emissions will be decreased
+- `I`: If CPU usage can be decreased the software intensity will be decreased
```

Comment:
I would argue I is carbon intensity and not software intensity. Thoughts?

Reply:
agree!